File: Data Structures & Algorithms/missing-number/submission-0.java
class Solution {
    public int missingNumber(int[] nums) {
        int result = 0;
        for (int i = nums.length; i > 0; i--) {
            result += i;
        }
        for (Integer i : nums) {
            result -= i;
        }
        return result;
    }
}

File: Data Structures & Algorithms/remove-element/submission-0.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        k = 0
        different = []
        
        for num in nums:
            if num != val:
                k += 1
                different.append(num)

        for i in range(k):
            nums[i] = different[i]

        return k

        